src/v/security/gssapi_authenticator.cc

```diff
@@ -208,6 +213,15 @@ ss::future<result<bytes>> gssapi_authenticator::authenticate(bytes auth_bytes) {
             _impl->reset();
             return principal;
         });
+        auto lifetime = co_await _worker.submit(
+          [this]() { return _impl->lifetime(); });
+        if (lifetime.has_value()) {
+            _session_expiry.emplace(
+              ss::lowres_system_clock::now() + lifetime.value());
```

Comment:
This is probably my fault, but I think the clock should be `lowres_clock` because it's `is_steady`.

Reply:
Good catch. Missed this while flipping back and forth between dealing with unix time on metrics diffs.